pp_netlib/network.py

```diff
@@ -1,19 +1,23 @@
+from functools import partial
+from multiprocessing import Pool
 import os, sys
 import numpy as np
 import pandas as pd
 import scipy
 
 from pp_netlib.functions import construct_with_graphtool, construct_with_networkx, summarise
+from pp_netlib.gt_prune import gt_clique_prune, gt_get_ref_graph
+from pp_netlib.nx_prune import alt_nx_get_clique_refs, nx_get_clique_refs, nx_get_connected_refs
```

Comment:
Double-check this doesn't unconditionally import graph-tool and/or networkx (as package should be usable with just one of these)

Reply:
moved imports to prune method to fix this